common/src/app/common/types/tokens_lib.cljc

```diff
@@ -1173,7 +1174,27 @@ Will return a value that matches this schema:
                                        (set sets))))
          lib' themes-data)
         lib')))
-
+  (decode-legacy-json [this parsed-legacy-json]
+    (let [other-data (select-keys parsed-legacy-json ["$themes" "$metadata"])
+          sets-data (dissoc parsed-legacy-json "$themes" "$metadata")
+          dtcg-sets-data (walk/postwalk
+                          (fn [node]
+                            (cond-> node
+                              (and (map? node)
+                                   (contains? node "value")
+                                   (sequential? (get node "value")))
+                              (update "value"
+                                      (fn [seq-value]
+                                        (map #(set/rename-keys % {"type" "$type"}) seq-value)))
```

Comment:
This condition is added to handle tokens like this:
https://github.com/penpot/penpot/pull/5695/files#diff-4179ab5570deb92b9019518866b7d9c6419bad5f7fe83ff91c6b7e20e068f944R715-R737